web_app/main.py

```diff
@@ -4,6 +4,7 @@
 from fastapi.templating import Jinja2Templates
 from sqlalchemy.orm import Session
 
+from __logger__ import logger
```

Comment:
It's better to use next construction:
` import logging`
`logger = logging.getLoger(__name__)`
It will add also `filename` to log what can improve debugging or looking for information